expyfun/_experiment_controller.py

```diff
@@ -442,9 +442,9 @@ def screen_text(self, text, pos=[0, 0], color='white', font_name='Arial',
         return scr_txt
 
     def screen_prompt(self, text, max_wait=np.inf, min_wait=0, live_keys=None,
-                      timestamp=False, clear_after=True, pos=[0, 0],
-                      color='white', font_name='Arial', font_size=24,
-                      wrap=True, units='norm', attr=True):
+                      click=False, timestamp=False, clear_after=True,
```

Comment:
new params should go at the end to avoid backward compat break with `args` usage